scripts/releaseChangelog.mjs

```diff
@@ -213,14 +222,39 @@ We'd like to offer a big thanks to the ${
 
 TODO INSERT HIGHLIGHTS
 ${changeLogMessages.length > 0 ? '\n\n' : ''}${changeLogMessages.join('\n')}
-### \`@mui/x-data-grid@v__VERSION__\` / \`@mui/x-data-grid-pro@v__VERSION__\` / \`@mui/x-data-grid-premium@v__VERSION__\`
-${logChangelogSection(dataGridCommits, '#### Changes')}
 
-### \`@mui/x-date-pickers@v__VERSION__\` / \`@mui/x-date-pickers-pro@v__VERSION__\`
-${logChangelogSection(pickersCommits, '#### Changes')}
+### \`@mui/x-data-grid@v__VERSION__\`
```

Comment:
This script would need slight adjustment to also generate the headings of a component family as well as a decrease in heading level for all the packages to h4.

Reply:
Thanks, I 'm doing it '